### PR TITLE
Graph tweaks

### PR DESCRIPTION
**Pull request by 3v0k4**
Feel free to review Rails and Elm separately.

* move `app/controllers/api/realtime/measurements_controller.rb#stream_measurements` to `app/controllers/api/measurements.rb#index` and tests
* remove some more Angular
* had to start using the pipeline Elm decoders cause there's no `map9` (https://package.elm-lang.org/packages/elm-lang/core/5.1.1/Json-Decode#mapping)

Haven't tested much Elm cause I want to wait a bit. I'm not sure the design won't change in the next few PRs

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -14,7 +15,7 @@ Endpoints
 ### Parameters
 
 | name           | type              | default value |
-|:---------------|:------------------|:--------------|
+| :------------- | :---------------- | :------------ |
```
Comment (gitjul) [inline doc/api.md]: 😍

File: doc/api.md [hunk 2]
```diff
@@ -162,15 +163,17 @@ curl http://aircasting.org/api/sessions/9586.json
 
 ## Measurements for stream id
 
-GET `/api/realtime/stream_measurements.json`
+GET `/api/measurements.json`
```
Comment (gitjul) [inline doc/api.md]: 👍

File: app/javascript/elm/src/Main.elm [hunk 3]
```diff
@@ -494,7 +538,7 @@ view model =
                                     "FixedSessionsMapCtrl"
                                 )
                             ]
-                            [ div [ class "sessions", attribute "ng-controller" "SessionsGraphCtrl" ]
```
Comment (gitjul) [inline app/javascript/elm/src/Main.elm]: 🎉

Comment (3v0k4) [reply app/javascript/elm/src/Main.elm]: YEAH!

**Comment by 3v0k4** [comment]
Thanks for taking a look @gitjul

**Pull request merged**